daml-assistant/daml-helper/src/DA/Daml/Helper/Main.hs

```diff
@@ -337,6 +338,14 @@ commandParser = subparser $ fold
         <> metavar "TOKEN_PATH"
         <> help "Path to the token-file for ledger authorization"
 
+    timeoutOption :: Parser TimeoutSeconds
+    timeoutOption = option auto $ mconcat
+        [ long "timeout"
+        , metavar "TIMEOUT"
+        , value 30
+        , help "Timeout of gRPC operations in seconds. Defaults to 30s"
```

Comment:
```suggestion
        , help "Timeout of gRPC operations in seconds. Defaults to 30s. Must be > 0."
```

Reply:
The reason I think this is worth mentioning is there are timeout APIs out there where `-1` or `0` have a special "unlimited" meaning.